### ISSUE TITLE
Can't post a checkbox value directly from within a form element.

### ISSUE THREAD

**Issue by arvanoid** [vuetifyjs/vuetify#838]
- What will it allow you to do that you can't do today?

Post a checkbox value directly from within a form element.

  - How will it make current work-arounds straightforward?

By including a hidden checkbox field into the DOM
```html
<input type="checkbox" name="my-test1" id="my-test1" tabindex="-1" value="true">
```
which will keep track of the current checkbox value.

**Comment by johnleider** [comment]
I am curious your particular use-case where you have to submit using a regular form.

**Comment by arvanoid** [comment]
Using it in a small form built with Laravel's Blade. Let's say a simple login form with a remember me option. It is easier to simply populate the form and then submit with a regular post avoiding ajax calls.

But then we'll also need a way to repopulate the checkbox without using v-model. This is possible with text fields but I can't find a way for checkboxes.

**Comment by johnleider** [comment]
The reason why, currently, is because text-fields are the only form component that create an actual input.

**Comment by arvanoid** [comment]
Yes :( think it is a very well needed feature. Do you intent to support direct population?

**Comment by johnleider** [comment]
This needs to be propagated to numerous components. Since 1.1 is focused on touching up that area of the framework anyways, I'm going to move this issue to that.

**Comment by arivera12** [comment]
All components are broken if you try to submits then using the default browser submit. The only that works is the text-field component. Select component submits the text instead of the value. There is no way to submit this fields correctly ajax is needed at all time. Submitting a form manually using ajax its a waste of time when you have ajax submit plugins.

**Comment by arivera12** [comment]
@johnleider any deadline for the release of vuetify 1.1 with the inputs refactor?

**Comment by nekosaur** [comment]
Please do not comment on closed issues. v1.1.0-beta.0 was released last week.
